standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -864,6 +867,98 @@ public void abortTxns(AbortTxnsRequest rqst) throws NoSuchTxnException, MetaExce
     }
   }
 
+  private void updateReplId(Connection dbConn, ReplLastIdInfo replLastIdInfo) throws SQLException, MetaException {
+    PreparedStatement pst = null;
+    ResultSet rs = null;
+    String lastReplId = quoteString(Long.toString(replLastIdInfo.getLastReplId()));
+    Statement stmt = dbConn.createStatement();
```

Comment:
createStatement call should be moved inside try block for close to get called in finally block.

Reply:
done